.packit.yaml

```diff
@@ -0,0 +1,14 @@
+specfile_path: python-requre.spec
+synced_files:
+  - python-requre.spec
+  - .packit.yaml
+# package project name what correspond to package name in setup.py
+upstream_package_name: requre
+# name what you used as name in specfile (for python packages it starts with python-)
+downstream_package_name: python-requre
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    targets: fedora-stable
```

Comment:
Maybe this:

```suggestion
    targets:
    - fedora-stable
```

Reply:
I've used original documentation for this: https://packit.dev/docs/configuration/#packit-service-jobs. So documentation has to be fixed